docs/resources/namespace.md

```diff
@@ -118,10 +116,9 @@ resource "temporalcloud_namespace" "terraform3" {
   regions        = ["aws-us-east-1"]
   api_key_auth   = true
   retention_days = 14
-
-  // Prevents Terraform from deleting namespace. Must remove this before destroying resource.
-  lifecycle {
-    prevent_destroy = true
```

Comment:
I know it's a little redundant, but i think we should keep this here too encourage the usage of both backend delete protection and terraform destroy protection

Reply:
IMHO its very confusing for someone reading this, two lifecycle fields.

Reply:
I wonder if there's a way to automatically set TF's version if ours is set?